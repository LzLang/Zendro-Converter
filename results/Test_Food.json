{
    "Hamburger": {
        "description": "Is a food consisting of fillings\u00e2\u20ac\u201dusually a patty of ground meat, typically beef\u00e2\u20ac\u201dplaced inside a sliced bun or bread roll.",
        "type": "String"
    },
    "HotDog": {
        "description": "Is a food consisting of a grilled or steamed sausage served in the slit of a partially sliced bun.",
        "type": "String"
    },
    "ApplePie": {
        "description": "Is a fruit pie in which the principal filling ingredient is apples",
        "type": "Boolean"
    }
}