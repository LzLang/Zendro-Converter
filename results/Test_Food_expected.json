{
	"Hamburger": {
		"description": "Is a food consisting of fillings—usually a patty of ground meat, typically beef—placed inside a sliced bun or bread roll.",
		"type": "string"
	},
	"HotDog": {
		"description": "Is a food consisting of a grilled or steamed sausage served in the slit of a partially sliced bun.",
		"type": "string"
	},
	"ApplePie": {
		"description": "Is a fruit pie in which the principal filling ingredient is apples",
		"type": "boolean"
	}
}